types/reactstrap/lib/Popper.d.ts

```diff
@@ -7,127 +7,130 @@
+        applyStyle?: BaseModifier & {
+            onLoad?: Function;
+            gpuAcceleration?: boolean;
+        };
+        computeStyle?: BaseModifier & {
+            gpuAcceleration?: boolean;
+            x?: "bottom" | "top";
+            y?: "left" | "right";
+        };
+        [name: string]: (BaseModifier & Record<string, any>) | undefined;
     }
 
     export interface Offset {
-      top: number;
-      left: number;
-      width: number;
-      height: number;
+        top: number;
+        left: number;
+        width: number;
+        height: number;
     }
 
     export interface Data {
-      instance: Popper;
-      placement: Placement;
-      originalPlacement: Placement;
-      flipped: boolean;
-      hide: boolean;
-      arrowElement: Element;
-      styles: CSSStyleDeclaration;
-      boundaries: Object;
-      offsets: {
-        popper: Offset,
-        reference: Offset,
-        arrow: {
-          top: number,
-          left: number,
-        },
-      };
+        instance: Popper;
+        placement: Placement;
+        originalPlacement: Placement;
+        flipped: boolean;
+        hide: boolean;
+        arrowElement: Element;
+        styles: CSSStyleDeclaration;
+        boundaries: Object;
+        offsets: {
+            popper: Offset;
+            reference: Offset;
+            arrow: {
+                top: number;
+                left: number;
+            };
+        };
     }
 
     export interface PopperOptions {
-      placement?: Placement;
-      positionFixed: boolean;
-      eventsEnabled?: boolean;
-      modifiers?: Modifiers;
-      removeOnDestroy?: boolean;
-      onCreate?(data: Data): void;
-      onUpdate?(data: Data): void;
+        placement?: Placement;
+        positionFixed: boolean;
+        eventsEnabled?: boolean;
+        modifiers?: Modifiers;
+        removeOnDestroy?: boolean;
+        onCreate?(data: Data): void;
+        onUpdate?(data: Data): void;
     }
 
     export interface ReferenceObject {
-      clientHeight: number;
-      clientWidth: number;
-      getBoundingClientRect(): ClientRect;
+        clientHeight: number;
+        clientWidth: number;
+        getBoundingClientRect(): ClientRect;
     }
-  }
+}
 
-  declare class Popper {
+declare class Popper {
     static modifiers: (Popper.BaseModifier & { name: string })[];
     static placements: Popper.Placement[];
     static Defaults: Popper.PopperOptions;
 
     options: Popper.PopperOptions;
 
-    constructor(reference: Element | Popper.ReferenceObject, popper: Element, options?: Popper.PopperOptions);
+    constructor(
+        reference: Element | Popper.ReferenceObject,
+        popper: Element,
+        options?: Popper.PopperOptions
+    );
 
     destroy(): void;
     update(): void;
     scheduleUpdate(): void;
     enableEventListeners(): void;
     disableEventListeners(): void;
-  }
+}
 
-  declare module 'popper.js' {
-    export default Popper;
-  }
+export default Popper;
```

Comment:
add `export as namespace Popper;` to make sure the module is still accessible as a global (which seems to be the case from the examples in https://github.com/FezVrasta/popper.js/).